includes/class-wc-payments-utils.php

```diff
@@ -992,4 +992,31 @@ public static function enqueue_style( $handle, $path = '', $deps = [], $version
 		}
 		wp_enqueue_style( $handle );
 	}
+
+	/**
+	 * Returns language data: english name and native name
+	 *
+	 * @param string $language Language code.
+	 *
+	 * @return array
+	 */
+	public static function get_language_data( $language ) {
+		require_once ABSPATH . 'wp-admin/includes/translation-install.php';
+
+		$transaltions = wp_get_available_translations();
```

Comment:
Nitpick: there's a typo in variable name.

Reply:
Good catch! Fixed.